build.proj

```diff
@@ -31,6 +31,9 @@
   <Import Project="$(ToolsDir)CodeCoverage.targets" Condition="Exists('$(ToolsDir)CodeCoverage.targets')" />
   <Import Project="$(ToolsDir)PerfTesting.targets" Condition="Exists('$(ToolsDir)PerfTesting.targets') and '$(Performance)' == 'true'"/>
 
+  <!-- Hook that can be used to insert custom build tasks to the build process such as setup and/or cleanup tasks -->
+  <Import Project="build.override.targets" Condition="Exists('build.override.targets')" />
+  
```

Comment:
Question only: does this mean the build.override.targets must exist in the same directory as the build.proj, and therefore there is one build.override.targets for our repo, or can each child project that references this build.proj have their own override.targets?